### DISCUSSION TITLE
Git portable

### DISCUSSION THREAD

**Discussion by Doriim** [community/community#117303] [New to GitHub] [answered]
Hi, I have installed git portable on my usb stick and I would like to ask how can I access my pc folders from bash terminal? If it even possible? Thanks.

**Comment by AlexanderFRT** [comment, marked as answer]
1. **Plug in USB Stick**: Insert the USB stick into a USB port on your PC.

2. **Navigate to Git Portable Directory**: Open the file explorer and navigate to the directory where Git Portable is installed on the USB stick.

3. **Launch Git Bash**: Locate the `git-bash.exe` or similar executable file within the Git Portable directory. Double-click on it to launch Git Bash.

4. **Identify Drive Letter**: Once Git Bash is open, identify the drive letter assigned to the USB stick. This is usually shown in the Git Bash prompt.

5. **Navigate to PC Folders**: Use the appropriate drive letter to navigate to PC folders. For example, if your PC's `Documents` folder is located at `C:\Users\YourUsername\Documents`, you would navigate to it using commands like:
   ```bash
   cd /c/Users/YourUsername/Documents

**Reply by Doriim** [reply]
It works. Thank you so much.

**Comment by Aboalihsh** [comment]
5
